pkg/cache/server/handler.go

```diff
@@ -58,8 +58,16 @@ func init() {
 // /shards/sapphire/clusters/system:sapphire/apis/apis.kcp.dev/v1alpha1/apiexports
 //
 // /shards/amber/clusters/system:amber/apis/apis.kcp.dev/v1alpha1/apiexports
+//
+// Note:
+// not all paths require to have a valid shard name,
+// as of today the following paths pass through: "/livez", "/readyz", "/healthz"
 func WithShardScope(handler http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		if path := req.URL.Path; path == "/livez" || path == "/readyz" || path == "/healthz" {
```

Comment:
maybe in the future it could be extended but for now these are the paths i know about